steeltoe-oss/MusicStore/src/MusicStoreUI/MusicStoreUI.csproj

```diff
@@ -27,27 +26,19 @@
     <PackageReference Include="Microsoft.AspNetCore" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="$(AspNetCoreVersion)" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.1.14" />
```

Comment:
Why not 3.1?

Reply:
part of the next step... there's a bunch of out of date stuff in the current state